test/cuda-arch/base/clang@10.0.1_nvcc_c++14_lassen.cmake

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+# other BLT Project Developers. See the top-level LICENSE file for details
+# 
+# SPDX-License-Identifier: (BSD-3-Clause)
+
+#------------------------------------------------------------------------------
+# Example host-config file for the blue_os cluster at LLNL, specifically Lassen
+#------------------------------------------------------------------------------
+#
+# This file provides CMake with paths / details for:
+#  C/C++:   Clang with GCC 8.3.1 toolchain
```

Comment:
Is the gcc@8.3.1 toolchain implicit? Should it be explicitly listed in this host-config?
If not, consider updating this comment to indicate that the gcc is implicit.